routing_optimization_over_server/cvrptw_benchmark_gehring_homberger.ipynb

```diff
@@ -0,0 +1,355 @@
+{
+ "cells": [
```

Comment:
Fix link to [cost_matrix_creation.ipynb](https://github.com/NVIDIA/cuopt-examples/blob/a0943c4ade5021eab5d07e482272dee1a3abd9ba/routing_optimization_over_server/cost_matrix_creation.ipynb) currently this doesn't exist in that path or may be remove the line.

Reply:
I think this notebook links to cvrptw_service_team_routing.ipynb and not cost matrix

Reply:
Oh ok, lets fix it that way.

Reply:
that link works